Python/Plugins/src/GeoModel.cpp

```diff
@@ -0,0 +1,206 @@
+// This file is part of the ACTS project.
+//
+// Copyright (C) 2016 CERN for the benefit of the ACTS project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+// Must be on top to avoid some conflict between forward declare and typedef
+// Needed until https://gitlab.cern.ch/GeoModelDev/GeoModel/-/merge_requests/351
+// is deployed
+// clang-format off
+#include <GeoModelRead/ReadGeoModel.h>
+// clang-format on
```

Comment:
I saw the MR is already merged. But I think it shouldn't be changed in this PR.

https://gitlab.cern.ch/GeoModelDev/GeoModel/-/merge_requests/351